cmake/modules/Finddpdk.cmake

```diff
@@ -0,0 +1,33 @@
+# Try to find spdk
+#
+# Once done, this will define
+#
+# DPDK_FOUND
+# DPDK_INCLUDE_DIR
+# DPDK_LIBRARIES
+
+find_path(DPDK_INCLUDE_DIR rte_config.h
+  PATH_SUFFIXES dpdk)
+find_library(DPDK_rte_eal_LIBRARY rte_eal)
+find_library(DPDK_rte_mempool_LIBRARY rte_mempool)
+find_library(DPDK_rte_ring_LIBRARY rte_ring)
```

Comment:
I think we could add custom dpdk lib directory:
if (EXISTS ${DPDK_TARGET})
  list(APPEND _dpdk_include_path "${DPDK_TARGET}/include")
  list(APPEND _dpdk_lib_path "${DPDK_TARGET}/lib")
endif()

find_path(DPDK_INCLUDE_DIR rte_eal.h NO_DEFAULT_PATH PATHS ${_dpdk_include_path})
find_library(DPDK_LIBRARY NO_DEFAULT_PATH NAMES  rte_eal PATHS ${_dpdk_lib_path})